python/src/diskann_bindings.cpp

```diff
@@ -104,8 +104,8 @@ struct DiskANNIndex {
                               const char     *indexFilePath,
```

Comment:
You can actually remove this entire function - it somehow crept in and never should have merged in.

The latest working build is in my fork @ https://github.com/daxpryce/DiskANN/tree/daxpryce/pybind_diskann - I've been wrestling with getting our build to work via github actions across the OS / Python3.7-3.11 matrix over there and have also made this change

Reply:
Removed the function in the latest commit.

Kindly let me know if there is any specific issue related to python build on Github Actions you would like me to look into.